lms/envs/common.py

```diff
@@ -157,6 +157,9 @@
     # Can be turned off if course lists need to be hidden. Effects views and templates.
     'COURSES_ARE_BROWSABLE': True,
 
+    # Can be turned off if course search input needs to be hidden. Effects templates.
+    'COURSES_ARE_SEARCHABLE': True,
```

Comment:
I went with giving the FEATURE switch that name, so that it matches the existing one you see above it.

If you think the name is not appropriate, please let me know.